cmd/kubeadm/app/cmd/join.go

```diff
@@ -152,6 +152,9 @@ func AddJoinConfigFlags(flagSet *flag.FlagSet, cfg *kubeadmapiext.NodeConfigurat
 	flagSet.StringVar(
 		&cfg.NodeName, "node-name", "",
 		"Specify the node name.")
+	flagSet.StringVar(
```

Comment:
Set ClusterName to "kubernetes" by default in L104.

`cfg := &kubeadmapiext. NodeConfiguration{ClusterName: "kubernetes"}`